SpatialGDK/Source/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -72,9 +97,8 @@ bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(FString& Error)
 		GetEngine()->ShutdownWorldNetDriver(GetWorldContext()->World());
 	}
 
-	// This will use the URL / Map that was setup by the server worker (which is loaded first).
-	// By not specifying a hostname the connection defaults to local.
 	FURL URL = WorldContext->LastURL;
+	URL.Host = "127.0.0.1";
```

Comment:
Needed to prevent the host from being empty and triggering the wrong type of travel in a PIE context.